components/admin/lessons/AdminLessonChallenges.tsx

```diff
@@ -156,15 +157,21 @@ export const AdminLessonChallenges: React.FC<LessonChallengesProps> = ({
       await alterChallenge(makeGraphqlVariable(options, { lessonId }))
       window.location.reload()
     } catch (err) {
-      throw new Error(err)
+      Sentry.captureException(err)
     }
   }
-
   const allChallenges = !challenges
     ? []
-    : challenges.map((challenge: Maybe<Challenge>, key: number) => (
-        <LessonChallenge challenge={challenge} alter={alter} key={key} />
-      ))
-
+    : challenges.map((challenge: Maybe<Challenge>, key: number) => {
+        if (challenge !== null) {
```

Comment:
Lesson type is rather loose so challenges could be null or array of nulls. First ternary operator checks for null, if condition inside map checks every challenge. Since we're sure that every single challenge is not null we can pass `challenge as Challenge` to `LessonChallenge` component instead of `Maybe<Challenge>`. 
This won't change anything as far as UX goes.